code/src/components/GetThoughts.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import { formatDistance } from 'date-fns';
+
+const GetThoughts = ({ getThoughts, handleLikeButtonOnClick }) => {
+  return (
+    <section className="thoughts wrapper">
+      {getThoughts.map((thought) => (
+        <div
+          className="thoughts card"
+          key={thought._id}>
+          <div className="thoughts-text">
+            <p>{thought.message}</p>
+          </div>
+          <div className="thoughts-likes">
+            <button
+              className="btn-like"
+              type="button"
+              onClick={() => handleLikeButtonOnClick(thought._id)}>
+              <span className="heart">💗</span>
+            </button>
+            <p className="likes-counter">x {thought.hearts}</p>
+            <p className="thoughts-time">{formatDistance(new Date(thought.createdAt), new Date())} ago</p>
```

Comment:
formatDistance doesn't add the suffix by default so how about pass "addSuffix" option to have "ago" automatically:
```
<p className="thoughts-time">
              {formatDistance(new Date(thought.createdAt), Date.now(), { addSuffix: true })}
            </p>
```
Or you also can import "moment" instead, as you can find more information about it in this link I have used [https://momentjs.com/](url):

npm install moment --save
import moment from 'moment'
```

    <p className="moment">
              {moment(thought.createdAt).fromNow()}
            </p>
```